src/Utils/DEM.jl

```diff
@@ -0,0 +1,77 @@
+
+
+abstract type DEM end
+
+############# DEM Interface begin ######################################
+
+"""
+    getCoordinate(dem::T,index::Tuple{Integer,Integer}) where T <: DEM
+
+    Get the coordinate for a certain index in the DEM. 
+    (All the DEM's are stored as a matrix of heights)
+"""
+function getCoordinate(dem::T,index::Tuple{Integer,Integer}) where T <: DEM
+     ErrorException("Not implemented for the following DEM type, $T")
+end
+
```

Comment:
What exactly is the functionality of this function.  Is it only added to give an error or is is because you want to implement the following DEM type later? It yes to the latter, it overkill to prepare for every type?

Reply:
It is to make a common interface for all DEM. I plan to do something similar for SAR Images. 

There should be a version of this function for all types of DEMs. Julia's multiple dispatches will automatically call the one that fits the DEM type. This version of the function just returns an error message in case someone adds a new DEM but forgets to add a version (overload) of this method.

It is not overkill but the way we will do inheritance.